packages/intl/src/message-loader.ts

```diff
@@ -98,7 +98,7 @@ export class MessageLoader {
     this.defaultLocale = defaultLocale;
 
     this._localeLoadingPromises = {};
-    this._parseCache = new Map();
+    this._parseCache = {};
```

Comment:
Using a plain object instead of a Map here might also give slightly better lookup performance in general. Probably negligible, or maybe even worse, though.